src/Microplate/Coordinates.php

```diff
@@ -50,7 +50,7 @@ public function __construct(string $row, int $column, CoordinateSystem $coordina
      * @param CoordinatesArray $coordinates
      * @param TCoord $coordinateSystem
      *
-     * @return static<TCoord>
+     * @return self<TCoord>
```

Comment:
Wir sollten in beiden Fällen `static` verwenden um ein Schlupfloch für ad-hoc Vererbung zu lassen.